### ISSUE TITLE
breaking change should bump minor if version pre-1.0

### ISSUE THREAD

**Issue by jasonkuhrt** [conventional-commits/conventionalcommits.org#165]
In accordance with https://semver.org/#spec-item-4

> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

I don't think it makes sense that conventional commits assume BREAKING CHANGE always means bumping the major. Some projects might be unstable for a while but still want to communication breaking changes to their users in a changelog.

I think the spec should be as follows:

When BREAKING CHANGE is present:

1. if package version is >= 1.x.x bump MAJOR
2. if package version is < 1.x.x and type indicates feature MINOR
3. if package version is < 1.x.x and type indicates patch PATCH

**Comment by damianopetrungaro** [comment]
We should specify that if it is a 0.x.x we may not bump it up.

If > 0.x.y then https://semver.org/#spec-item-8

**Comment by jasonkuhrt** [comment]
> we may not bump it up.

It should probably not be ambiguous so `must not` rather than `may not`.

But then, how does a user bump to 1.0.0?

**Comment by damianopetrungaro** [comment]
When it is stable you tag it as 1.0.0 usually

**Comment by jasonkuhrt** [comment]
Yes certainly, but I was referring to the ability for conventional commits to support a well known commit pattern that would facilitate machines performing the bump. But for the ROI on automation around a singular event (to a project) is questionable.

**Comment by damianopetrungaro** [comment]
@jasonkuhrt wanna open a PR about this?
It sounds good to me :)

**Comment by jasonkuhrt** [comment]
@damianopetrungaro Will do!
